utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
+        if (link) {
+          setSkipLink(link);
+        }
+      });
+      // @ts-ignore
+      if (!observer) setObserver(obs);
+    }
+  };
+
+  const setSkipLink = (/** @type {HTMLElement} */ link) => {
+    // Update skip link
+    link.onclick = (e) => {
+      e.preventDefault();
+      // Focus on search input
+      const content = document.getElementById('searchInput');
+      content?.focus();
+    };
+    // Disconnect observer when done
+    // @ts-ignore
+    if (observer) observer.disconnect();
```

Comment:
Maybe it should also set observer to null after it disconnects so that a new one can be tracked when it's created?